test/fixtures/geoip-fixture/src/main/java/fixture/geoip/GeoIpHttpFixture.java

```diff
@@ -106,15 +107,25 @@ protected void after() {
     }
 
     private void copyFiles() throws Exception {
-        Files.copy(GeoIpHttpFixture.class.getResourceAsStream("/geoip-fixture/GeoLite2-ASN.tgz"), source.resolve("GeoLite2-ASN.tgz"));
-        Files.copy(GeoIpHttpFixture.class.getResourceAsStream("/geoip-fixture/GeoLite2-City.mmdb"), source.resolve("GeoLite2-City.mmdb"));
+        Files.copy(
```

Comment:
Why was this not needed before? Are we not using clean working directories for these tests anymore?

Reply:
I honestly don't know why it's not happening in Jenkins, but it happens on my laptop, so it's not exclusive to Buildkite. The buildkite workspace starts with a clean checkout.

Reply:
Yeah, this strikes me as odd. We should be creating a unique temp directory on every initialization here. If we are overwriting files that means we aren't, or `Files.createTempDirectory` isn't doing what I think it should. My fear is we are using common directories here we risk running into a race condition where one fixture tries to blow away the files of another one mid-test execution.

Reply:
See #99389